projects/cm_mcu/LocalTasks.c

```diff
@@ -34,6 +34,9 @@
 // local prototype
 void Print(const char *str);
 
+uint32_t ff_PRESENT_mask = 0xff;
```

Comment:
Not sure I understand this choice of initial variable? it seems like you are setting 'present' for the bottom 8 bits and 'not present' for the rest?

Reply:
I agree. It was my bad. This should have been 0x0 to be a better choice that no bits are initially enabled.

Reply:
Done.